services/core/shared/core/dynamicFees.js

```diff
@@ -326,7 +326,7 @@ const getEstimateFeeByte = async () => {
 		return { data: { error: `Action not supported for Lisk Core version: ${getCoreVersion()}.` } };
 	}
 
-	const latestBlock = getLastBlock();
+	const latestBlock = (await getBlocks({ limit: 1 })).data[0];
```

Comment:
Why do we need this? Isn't the last received block sufficient?

Reply:
At times, `getLastBlock` returns to be `undefined`, thus breaking the fee estimation algorithm. Hence, updated it to the promise-based invocation as it is much more reliable.